TheMovieProject/Cells/UpcomingCollectionViewCell.swift

```diff
@@ -8,8 +8,9 @@
 import UIKit
 
 class UpcomingCollectionViewCell: UICollectionViewCell {
-    
+
     @IBOutlet weak var MovieDuration: UILabel!
-    @IBOutlet weak var MovieLabel: UILabel!
     @IBOutlet weak var MoviePoster: UIImageView!
```

Comment:
IBOutlet isimlendirmesinde label ile yazdıkların gayet iyi, poster için de image olduğunu belirtecek bir şey aradı gözlerim. Bir sürü IBOutlet olduğunda değişkenlerin arasında kaybolmamak adına "moviePosterImageView" gibi bir isimlendirme daha iyi olabilir gibi. Küçük bir değişiklik, bu projede çok önem kazanacak bir şey değil ama daha büyük çaplı bir proje için düşünülmesinde fayda var gibi